bitacross-worker/scripts/litentry/release/template/relay-alice.service

```diff
@@ -5,7 +5,7 @@ Description=Litentry Relaychain Alice
 Type=simple
 User=USER
 WorkingDirectory=/opt/litentry/parachain
-ExecStart=/opt/litentry/parachain/polkadot --base-path /opt/litentry/parachain/relay-alice --chain /opt/litentry/parachain/rococo-local-chain-spec.json --alice --port 30336 --ws-port 9946 --rpc-port 9936
+ExecStart=/opt/litentry/parachain/polkadot --base-path /opt/litentry/parachain/relay-alice --chain /opt/litentry/parachain/rococo-local-chain-spec.json --alice --port 30336 --rpc-port 9946
```

Comment:
I noticed there are quite some places that rpc-port change from 9936 to 9946. But old ws-port 9946 is just removed. Can we simply remove `--ws-port 9946` and not touch `--rpc-port 9936` ?  Any special consideration about port number?

Reply:
As spoken it replaces the old `ws-port`